trin-core/src/portalnet/overlay_service.rs

```diff
@@ -643,27 +643,42 @@ impl<TContentKey: OverlayContentKey + Send, TMetric: Metric + Send>
         }
     }
 
-    /// Attempts to build a `Accept` response for a `Offer` request.
+    /// Attempts to build a `Accept` response for an `Offer` request.
```

Comment:
```suggestion
    /// Attempts to build an `Accept` response for an `Offer` request.
```
the nittiest nit-pick